src/App.css

```diff
@@ -36,3 +36,56 @@
     transform: rotate(360deg);
   }
 }
+
+.sidebarStyle {
+  display: inline-block;
+  position: absolute;
+  top: 80px;
+  right:0;
+  margin: 12px;
+  background-color: #404040;
+  color: #ffffff;
+  z-index: 1 !important;
+  padding: 6px;
+  font-weight: bold;
+}
+
+.mapContainer {
```

Comment:
This way of declaring classes would pollute global namespace.
CSS should be component-scoped . `withStyles()` can be used for component level stylng.